htdocs/mri/jiv/get_file.php

```diff
@@ -136,10 +136,10 @@
     $MimeType         = 'image/vnd.nrrd';
     $DownloadFilename = basename($File);
     break;
-case 'DICOMTAR':   
+case 'tar':   
```

Comment:
@MounaSafiHarab - good improvement to just handle all Tars.  
- if using 'tar' over DICOMTAR, let's also remove the code at that sets $FileExt = 'DICOMTAR'
- I specified dicomtar, envisioning that Media Module etc might store other tars of stimulus files, or other non-imaging tars stored elsewhere (even genetics).  i.e. Will all future tars in Loris be images accessible on the Imaging path?